web/src/components/itemList.js

```diff
@@ -29,17 +29,20 @@ class ItemList extends Component {
         this.rate = 0.001;
     }
 
-    itemClicked = event => {
+    itemPlus = event => {
         const idx = event.target.getAttribute('id');
-        if(this.selected[idx]) {
-            this.props.removeItem(this.state.items[idx].price);
-            event.target.setAttribute('variant', 'primary')
-        }else {
-            this.props.addItems(this.state.items[idx].price);
-            event.target.setAttribute('variant', 'secondary')
-        }
+        this.props.addItems(this.state.items[idx].price);
+        this.selected[idx] += 1;
+    }
+
+    itemMinus = event => {
+        const idx = event.target.getAttribute('id');
+        this.props.removeItem(this.state.items[idx].price);
+        this.selected[idx] -= 1;
+    }
 
-        this.selected[idx] = !this.selected[idx];
+    itemNumber = event => {
+        // 구현 예정
```

Comment:
If you want to leave a comment for the future works, the general format would be like this: `// todo (your name): make ~~ function`